### ISSUE TITLE
Could not decompress chromedriver_linux64.zip 

### ISSUE THREAD

**Issue by Andrew-Max** [titusfortner/webdrivers#18]
```
   RuntimeError:
        Could not decompress chromedriver_linux64.zip to get /home/me/.webdrivers/chromedriver
      # /home/me/.rvm/gems/ruby-2.4.1/gems/webdrivers-3.3.1/lib/webdrivers/common.rb:63:in `download'
```

During the middle of the day today, all of my selenium driven tests using chromedriver started failing with the above error.

I'm really not sure how to figure out whether this error is at the `webdrivers` level or, the `chromedriver-helper` level, or at their interface but it seems like the `webdrivers` gem is unable to unzip some file that it expects the chromedriver-helper to have placed at `~/.chromedriver-helper/70.0.3538.16/linux64`.

Please let me know if there is any more info I can provide that would be helpful as I know this is a bit vague.

**Comment by twalpole** [comment]
This is because a `70.0.3538.16` directory was added to chromedriver download location today for some reason - https://chromedriver.storage.googleapis.com/index.html .  It would probably be better to read from the LATEST_RELEASE file rather than depending on directory name for determine the latest version.

**Comment by ibrahima** [comment]
That is indeed the case - see https://github.com/flavorjones/chromedriver-helper/issues/62 for a response from some Google engineers. It looks like they've pushed a beta version up but expected people to use the LATEST_RELEASE file to get the stable version.

**Comment by Andrew-Max** [comment]
Resoled  as mentioned by @twalpole. Thanks for the quick responses!

**Comment by Andrew-Max** [comment]
I'm guessing people may be aware of this but this just rebroke for me with a new error: 

```
     NoMethodError:
       undefined method `[]' for nil:NilClass
     # /home/me/.rvm/gems/ruby-2.4.1/gems/webdrivers-3.3.1/lib/webdrivers/chromedriver.rb:12:in `current'

```

**Comment by rossta** [comment]
@Andrew-Max I believe upgrading to the most recent webdrivers (3.4.1) and deleting any local downloads of the experimental chromedriver version will fix the issue. See [this comment](https://github.com/titusfortner/webdrivers/issues/19#issuecomment-422419375) in #19.

**Comment by Andrew-Max** [comment]
@rossta so I should delete  the `/70.0.3538.16` directory at `~/.chromedriver-helper/70.0.3538.16/` ?

updating to `3.4.1` alone did not resolve it.

**Comment by rossta** [comment]
@Andrew-Max Yes, as the linked comment suggests, but in the `~/.webdrivers` directory.

I don't know about the behavior of using both webdrivers and chromedriver-helper at the same time, but I'd probably delete both if they exist since 70 is still an experimental version.

**Comment by nicolascian** [comment]
Upgrade to 3.4.1 and then force to download the driver again

```
require 'webdrivers'
Webdrivers.logger.level = :debug
Webdrivers::Chromedriver.download
2018-09-18 14:41:41 DEBUG Webdrivers Found Site: http://chromedriver.storage.googleapis.com
2018-09-18 14:41:41 DEBUG Webdrivers Versions now located on downloads site: [...]
2018-09-18 14:41:42 DEBUG Webdrivers Successfully downloaded chromedriver_linux64.zip
2018-09-18 14:41:42 DEBUG Webdrivers Decompressing zip
2018-09-18 14:41:42 DEBUG Webdrivers Decompression Complete
2018-09-18 14:41:42 DEBUG Webdrivers Deleting chromedriver_linux64.zip
2018-09-18 14:41:42 DEBUG Webdrivers Completed download and processing of /home/nicolas/.webdrivers/chromedriver
```

**Comment by titusfortner** [comment]
Fixed this with #20 & 3.4.2 release
Thanks.
